adaptive_breakpoints/README.md

```diff
@@ -0,0 +1,97 @@
+# adaptive_breakpoints
+
+The `adaptive_breakpoints` package for Flutter allows you to use the adaptive [breakpoints entries](https://material.io/design/layout/responsive-layout-grid.html#breakpoints) from the [Material Design System](https://material.io/).
+
+## Getting Started
+
+This package provides the material breakpoint entries for adaptive/responsive development in Flutter.
+
+First, add the `adaptive_breakpoints` package to your pubspec dependencies.
+
+To import `adaptive_breakpoints`:
+
+```dart
+import 'package:adaptive_breakpoints/adaptive_breakpoints.dart
+```
+
+Here is an example of how to implement the Adaptive Breakpoint entries in Flutter Development.
+
+In this example, we created an adaptive container which will 'activate' anytime that window screen is
+on same as the window limit. If they are the same the child will show otherwise it will be an empty container.
+
+Try it yourself with this [dartpad](https://dartpad.dev/be6d3b390933220daf14bd4afd7e438c)
+
+```dart
+import 'package:flutter/material.dart';
+import 'package:adaptive_breakpoints/adaptive_breakpoints.dart';
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        body: Column(
+          children: [
+            AdaptiveContainer(
+              windowLimit: AdaptiveWindowType.xs,
+              child: Text('This is a extra small window'),
+            ),
+            AdaptiveContainer(
+              windowLimit: AdaptiveWindowType.s,
+              child: Text('This is a small screen'),
```

Comment:
Be consistent with using window or screen but not both. 

```suggestion
              child: Text('This is a small window'),
```

Reply:
Done